faunadb/query.go

```diff
@@ -1918,6 +1918,46 @@ func Trunc(value interface{}, options ...OptionalParameter) Expr {
 	return fn1("trunc", value, options...)
 }
 
+// Count returns the number of elements in the collection.
+//
+// Parameters:
+// collection Expr - the collection
+//
+// Returns:
+// a new Expr instance
+//
+// See: https://docs.fauna.com/fauna/current/api/fql/functions/count
+func Count(collection ...interface{}) Expr {
```

Comment:
I think using a varargs here could be dangerous, it allow the user to do `Call(Match(...), Match(...))` which is not allowed